bokehjs/src/lib/models/callbacks/open_url.ts

```diff
@@ -36,15 +37,15 @@ export class OpenURL extends Callback {
       window.open(url)
   }
 
-  execute(_cb_obj: unknown, {source}: {source: ColumnarDataSource}): void {
+  execute(_cb_obj: unknown, {source, behavior}: {source: ColumnarDataSource, behavior: TapBehavior}): void {
```

Comment:
```suggestion
  execute(_cb_obj: unknown, {source, behavior="select"}: {source: ColumnarDataSource, behavior?: TapBehavior}): void {
```
Alternatively update new tests in `unit/models/callbacks/openurl.ts`.